tests/arq/src/test/resources/openjdk18-mysql-persistent-s2i.json

```diff
@@ -0,0 +1,639 @@
+{
```

Comment:
should not we use templates form application templates? or how this will be synced?

Reply:
We do provide a single application template for the openjdk image, see: https://github.com/jboss-openshift/application-templates/tree/225f292/openjdk This template is used in other test.

Reply:
ok.. this seems reasonable